custom-images/.env

```diff
@@ -12,6 +12,7 @@ LOCAL_DATA_PATH=.\docker\data
 CD_HOST=cd.dockerexamples.localhost
 CM_HOST=cm.dockerexamples.localhost
 ID_HOST=id.dockerexamples.localhost
+AH_HOST=ah.dockerexamples.localhost
```

Comment:
Let's name it as HRZ_HOST.  Authoring host is the internal name.

Reply:
Done